tests/plugins/range.test.ts

```diff
@@ -124,6 +124,62 @@ describe("range plugin", () => {
       });
     });
 
+    describe("create a range and remove multiple columns", () => {
+      beforeEach(() => {
```

Comment:
I seems that all those test were already passing before your fix, it would be nice to have a test or two that tests the fix. For this you'll need to adapt `PluginTestRange`